### PR TITLE
Dashboard manager breaking change announcement.

### PR DESCRIPTION
**Pull request by jdre-c8y**

### CODE REVIEW

**Review by rahulc8y — APPROVED** [review]

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-date: 2024-04-26
-title: Reports page extracted into a separate plugin
+date: 2024-12-10
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md]: @rahulc8y Why do we overwrite the first announcement? Should we keep it and create a new one?

Comment (jdre-c8y) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-reports-module-as-separate-plugin.md]: oh, no, it's my mistake, I wanted to copy reports announcement and use it as template for new announcement. it should be new one.

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-dashboard-manager-as-separate-plugin.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: no-issue
+version: 1021.0.0
+---
+In the upcoming version the dashboard manager module is extracted from the Cockpit application and added as a separate plugin.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-dashboard-manager-as-separate-plugin.md]:
```suggestion
In an upcoming version the dashboard manager module will be extracted from the Cockpit application and added as a separate plugin.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-dashboard-manager-as-separate-plugin.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: no-issue
+version: 1021.0.0
+---
+In an upcoming version the dashboard manager module will be extracted from the Cockpit application and added as a separate plugin.
+This change might be a breaking change as the dashboard manager module will no longer be part of `@c8y/ngx-components/context-dashboard`, but will be part of `@c8y/ngx-components/dasboard-manager`.
+
+This change does not affect the layout and UX of the Cockpit application nor does it add any new functionalities.
+As a side effect, a performance improvement might be noticed as dashboard manager from now on will be loaded lazily (that is, only after main application is loaded).
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-dashboard-manager-as-separate-plugin.md]:
```suggestion
As a side effect, a performance improvement might be noticed as the dashboard manager from now on will be loaded lazily (that is, only after main application is loaded).
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**